src/IssueViz.UnitTests/TableControls/IssueTablesSelectionMonitorTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarLint.VisualStudio.Integration;
+using SonarLint.VisualStudio.Integration.UnitTests;
+using SonarLint.VisualStudio.IssueVisualization.TableControls;
+
+namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.TableControls
+{
+    [TestClass]
+    public class IssueTablesSelectionMonitorTests
+    {
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
```

Comment:
Interesting. I'll add that to my future classes as well.

Reply:
If I remember correctly, the test helper is pretty smart.
You give it two lists: optional imports and required imports.
It will check that composition fails if any required imports are missing, and that composition succeeds if any optional import is missing.